src/lib/CMakeLists.txt

```diff
@@ -561,6 +563,8 @@ set(
     utils/singleton.hpp
     utils/string_utils.cpp
     utils/string_utils.hpp
+    utils/sqlite_wrapper.cpp
```

Comment:
Does anyone have a better idea on where to put this and check_table_equal so that both tests and benchmarks can use it?

Reply:
and do not forget the playground!

yet, no one liked my idea of a libHyriseAux.... :disappointed: 
Maybe just a `src/lib/auxiliary/` directory? it's not like libhyrise will ever be used stand-alone...

Reply:
Cue discussions on what belongs into `utils` and what belongs into `aux`. ;)

For now, I am tempted to just leave it in `utils` and save us the trouble of a more complicated project structure.

Reply:
Fine with me!